src/sidebars/star/star.html

```diff
@@ -1,2 +1,20 @@
 ﻿<img class="star" ng-src="assets/png/{{starCtrl.favouriteUrl()}}.png" ng-click="starCtrl.click(); $event.stopPropagation();"
      ng-mouseenter="starCtrl.mouseEnter()" ng-mouseleave="starCtrl.mouseLeave()" draggable="false" />
+
+<div class="confirmation-backdrop " ng-show="starCtrl.confirmationShow" ng-click="starCtrl.hideModal()">
```

Comment:
Maybe swap out `ng-show` for `ng-if` so all the confirm boxes aren't lingering in the DOM.